clean/new_orleans_pd_uof.py

```diff
@@ -32,14 +32,8 @@ def create_uof_lists(row: pd.Series):
         [element for list_ in uof_series for element in list_],
         columns=["tracking_number"] + uof_columns,
     )
-    df = df.drop(
-        columns=[
-            "use_of_force_type",
-            "use_of_force_level",
-            "use_of_force_effective",
-            "officer_injured",
-        ]
-    )
+
+    df = df.drop(columns=uof_columns)
     df = pd.merge(df, uof_df, on="tracking_number", how="outer")
```

Comment:
@pckhoi - I'm not sure as to if this is the correct way to re-join the `uof_df` with the original `df` after splitting the rows with multiple uof values.

Reply:
Shouldn't the `use_of_force_*` columns be in the `uof_officers` table? They are specific to each officer are they not?

Reply:
I added all of the `use_of_force_*` columns to the `uof_officers` table. 

I also added the `division`, `division_level` and `unit` columns to the `uof_officers` table because they are data specific to each officer.

The `use_of_force_reason`, `division`, `division_level`, and `unit` do not need to be split, which is why I have written this function as I did. Is there a more efficient way?

Reply:
Sorry, I did not notice that `use_of_force_reason` does not need to be split. My feelings is to simply disregard the column meaning when it comes down to what columns should end up in `uof_officers`. Let's decide simply on whether the column is splittable or not (and they certainly shouldn't be citizen columns). The `tracking_number` and/or `uof_uid` are the only non-splittable columns that need to also be in `uof_officers`, for that's how we track relationships between `use_of_force` and `uof_officers`. I would argue that we only need `uof_uid` on the `uof_officers` table even though there's no harm to also have `tracking_number` in there.

So I would put `use_of_force_reason`, `division`, `division_level`, and `unit` back into the `use_of_force` table unless you have reasons to believe they might be different for each officer in another dataset. Sorry for not being clear here.

Reply:
Got it! No worries at all. I went back and forth internally on this question. Glad it's cleared up. I'll make the changes.

Reply:
Q. @pckhoi I initially generated a `uof_uid` for the `use_of_force` table because we are extracting the `occur_date` for the `uof_incident` event from the `use_of_force` table. Instead generating a `uof_uid` for the uof_officers table will leave these events without an identifier. Thoughts?

Reply:
You can just generate `uof_uid` for the `use_of_force` table, then copy it to the `uof_officers` table. They have to be the same otherwise you cannot link the tables.

Reply:
Got it. Is there a downside to also copying the `uof_uid` to the `citizens_table`? Or Is the `tracking_number` a sufficient link between the `use_of_force` table and the `uof_citizens` table?

Reply:
Do you need `tracking_number` on the `uof_citizens` and `uof_officers`? If not then I recommend include only `uof_uid`. That's our id for each use of force record that we can guarantee to always be present and always have the same meaning no matter what the original dataset look like.

Reply:
Makes perfect sense. I have only included `uof_uid`. Please review the latest changes at your convenience. Thanks!